src/widgets/_index.scss

```
@@ -14,6 +14,9 @@ messagedialog,
 // Placeholder text in entries. Doesn't respect alpha
 @define-color placeholder_text_color #{"" + mix($fg-color, bg-color(1), $weight: 75%)};
 
+// Separator color
+@define-color menu_separator #{rgba(black, 0.15)};
```

You don't need to unquote this

```suggestion
@define-color menu_separator rgba(black, 0.15);
```